components/admin_console/team_channel_settings/channel/details/channel_moderation.tsx

```diff
@@ -0,0 +1,331 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import {isNil} from 'lodash';
+import classNames from 'classnames';
+import {ChannelModeration as ChannelPermissions} from 'mattermost-redux/types/channels';
+
+import FormattedMarkdownMessage from 'components/formatted_markdown_message';
+
+import {t} from 'utils/i18n';
+
+import AdminPanel from 'components/widgets/admin_console/admin_panel';
+import CheckboxCheckedIcon from 'components/widgets/icons/checkbox_checked_icon.jsx';
+t('admin.channel_settings.channel_moderation.title');
+t('admin.channel_settings.channel_moderation.subtitle');
+t('admin.channel_settings.channel_moderation.permissions');
+t('admin.channel_settings.channel_moderation.guests');
+t('admin.channel_settings.channel_moderation.members');
```

Comment:
I feel like this block does nothing - maybe you used it for debugging something but i think it can be removed?

Reply:
That's to flag those strings as being used by the translation tooling. `t` is usually called inline like it is in the `formattedMessages` below or not needed in cases like when you're using `FormattedMessage` or `Utils.localizeMessage`. Maybe these should be moved to be called when we pass the IDs into the `AdminPanel` component below?

Reply:
@fm2munsh @hmhealey 
The reason I have done this is because what I had originally:
![image](https://user-images.githubusercontent.com/17804942/75080726-cef52f00-54da-11ea-8bd9-c58b5c627997.png)
causes my file translations to be deleted when i do a `make i18n-extract`. I don't know why but I realised if I add those, it doesn't get deleted. Suggestions on how to fix it? 
After I run `make i18n-extract` with above photo and the `t(...)` deleted from the file.
![image](https://user-images.githubusercontent.com/17804942/75080781-01069100-54db-11ea-8062-4719bc3e2764.png)

Reply:
Update! Fixed by adding them to the `formattedMessages` object. I just referenced them through the formattedObject in the places I have a red square in the above photo.